README.md

```diff
@@ -8,9 +8,10 @@ brindando los conociminetos básicos, que posibiliten la comprensión de lo que
      1. ¿Qué es Visual Basic?
 2. [Variables]()
 3. [Constantes]()
-4. [Contadores]()
-5. [Acumuladores]()
-6. [Condionales]()
-7. [Ciclos]()
+4. Una**funcion constante** es una funcion lineal por la cual el rango no cambio sin importar cual miembro del dominio es usado . En progrmacion  la constante es el valor  que no puede ser alterado /modificado durante la ejecucion de un programa, unicamente puede ser leido  y corresponde a una longitud fija  de un area reservada en la memoria del ordenador . **tipos de constantes**constante predefinidas :es aquella que se selecciona por nombre y aparecen subrayadas por defecto en el editor de metodos y en la ventana del depurador ![ejemplo](https://doc.4d.com/4Dv16/picture/2292047/pict2292047.EN.png"onload").**constante literal**esta formada por una secuencia de caracteres que durante la ejecucion de un progrma  no cambia su valor  ejemplo "HOLA MUNDO/N"las constantes literales pueden ser de cuatro tipo de  **datos**:consttes de cadenas son las estan incluido entre comillas dobles(") y el compilador de D les asigna automaticamente  el tipo de string constantes numericas estan conformadas por numeros,un signo menos(-) y un punto decimal opcional **ejemplo**5.0,6,-5.1A,3.4 son algunos ejemplos de constantes numericas no validas  constantes fecha:se encuentra entre signos de administracion(!...) **ejemplo**!1997-01-01! constantes horas:se encuentra entre signos de interrogacion (??) se ordena hora:minutos:segundos.
```

Comment:
Good Job, but I want that you create a new folder and inside build the concept.

Remember using yourself words